.github/workflows/lint.yml

```diff
@@ -1,6 +1,6 @@
 name: Lint
 on:
-  pull_request:
+  # pull_request:
```

Comment:
Readd?

Reply:
Commented out temporarily while WIP to avoid wasting credits. Will revisit before merging.